terraform/gcp/projects/pilot-hubs.tfvars

```diff
@@ -35,6 +35,15 @@ notebook_nodes = {
     resource_labels : {
       "community" : "climatematch"
     }
+  },
+  "neurohackademy" : {
+    # We expect around 120 users
```

Comment:
Can you add a comment linking to the issue

Reply:
Done!